core/src/test/resources/hello_goodbye_papiv2.json

```diff
@@ -0,0 +1,242 @@
+        "runtimeAttributes": {
+          "bootDiskSizeGb": "10",
+          "continueOnReturnCode": "0",
+          "cpu": "1",
+          "cpuMin": "1",
+          "disks": "local-disk 10 SSD",
+          "docker": "ubuntu@sha256:71cd81252a3563a03ad8daee81047b62ab5d892ebbfbf71cf53415f29c130950",
+          "failOnStderr": "false",
+          "maxRetries": "0",
+          "memory": "2 GB",
+          "memoryMin": "2 GB",
+          "noAddress": "false",
+          "preemptible": "0",
+          "zones": "us-central1-b"
+        },
+        "shardIndex": -1,
+        "start": "2020-12-21T23:25:45.716Z",
+        "stderr": "gs://cloud-cromwell-dev-self-cleaning/cromwell_execution/ci/wf_hello/9b3f9d6d-67a9-405f-a55c-70550af02127/call-hello/stderr",
+        "stdout": "gs://cloud-cromwell-dev-self-cleaning/cromwell_execution/ci/wf_hello/9b3f9d6d-67a9-405f-a55c-70550af02127/call-hello/stdout"
+      }
+    ]
+  },
+  "end": "2020-12-21T23:25:53.577Z",
+  "id": "9b3f9d6d-67a9-405f-a55c-70550af02127",
+  "inputs": {},
+  "labels": {
+    "cromwell-workflow-id": "cromwell-9b3f9d6d-67a9-405f-a55c-70550af02127"
+  },
+  "metadataSource": "Archived",
+  "outputs": {
+    "wf_hello.goodbyes": "Goodbye there.",
+    "wf_hello.hellos": "Hello there!"
+  },
+  "start": "2020-12-21T23:25:43.044Z",
+  "status": "Succeeded",
+  "submission": "2020-12-21T23:25:42.461Z",
+  "submittedFiles": {
+    "inputs": "{}",
+    "labels": "{}",
+    "options": "{\n\n}",
+    "root": "",
+    "workflow": "version 1.0\n\ntask hello {\n    input {\n        String addressee\n    }\n    command {\n        echo \"Hello ~{addressee}!\"\n    }\n    output {\n        String salutation = read_string(stdout())\n    }\n    runtime {\n        docker: \"ubuntu@sha256:71cd81252a3563a03ad8daee81047b62ab5d892ebbfbf71cf53415f29c130950\"\n    }\n}\n\ntask goodbye {\n    input {\n        String addressee\n    }\n    command {\n        echo \"Goodbye ~{addressee}.\"\n    }\n    output {\n        String salutation = read_string(stdout())\n    }\n    runtime {\n        docker: \"ubuntu@sha256:71cd81252a3563a03ad8daee81047b62ab5d892ebbfbf71cf53415f29c130950\"\n    }\n}\n\n\nworkflow wf_hello {\n\n    call hello { input: addressee = \"there\" }\n    call goodbye { input: addressee = \"there\" }\n\n    output {\n        String hellos = hello.salutation\n        String goodbyes = goodbye.salutation\n    }\n}\n",
+    "workflowUrl": ""
+  },
+  "workflowName": "wf_hello",
+  "workflowProcessingEvents": [
+    {
+      "cromwellId": "cromid-5b21691",
+      "cromwellVersion": "55-8d04562-SNAP",
+      "description": "Finished",
+      "timestamp": "2020-12-21T23:25:53.577Z"
+    },
+    {
+      "cromwellId": "cromid-5b21691",
+      "cromwellVersion": "55-8d04562-SNAP",
+      "description": "PickedUp",
+      "timestamp": "2020-12-21T23:25:43.040Z"
+    }
+  ],
+  "workflowRoot": "gs://cloud-cromwell-dev-self-cleaning/cromwell_execution/ci/wf_hello/9b3f9d6d-67a9-405f-a55c-70550af02127/"
+}
```

Comment:
nitpicky: newline

Reply:
this was just a jq massaging of raw metadata a la `curl http://my-cromwell/v1/workflow-id/metadata | jq . > workflow-id.json` that wasn't really intended to be read by humans...